Recrutify/Recrutify.DataAccess/Repositories/ScheduleRepository.cs

```diff
@@ -44,5 +68,12 @@ public Task<Schedule> GetByDatePeriodAsync(Guid userId, DateTime date, int daysN
                                                     y.AvailableTime < date.Date.AddDays(daysNum)),
                         });
         }
+
+        private IFindFluent<Schedule, IEnumerable<ScheduleSlot>> GetFindFluentScheduleSlotsOfDatePeriod(FilterDefinition<Schedule> filter, DateTime periodStartDate, DateTime periodFinishDate)
```

Comment:
If you call this method one time, you can remove it and write same in public repository method

Reply:
+